atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/TracingPrefsConfig.java

```diff
@@ -80,7 +80,7 @@ public void run() {
                 loadedConfig = false;
             }
         } catch (Throwable t) {
-            log.error("Error occurred while refreshing {}: {}", TRACING_PREF_FILENAME, t, t);
+            log.error("Error occurred while refreshing {}", TRACING_PREF_FILENAME, t);
```

Comment:
You've already changed the logline - why not make this and the other one `unsafe` args?